src/components/modal/LoadingOverlay.js

```diff
@@ -53,7 +53,7 @@ const LoadingOverlay = ({ title, ...props }) => (
     <Overlay>
       <Centered zIndex={2}>
         <ActivityIndicator />
-        {title && <Title>{title}</Title>}
+        {title ? <Title>{title}</Title> : null}
```

Comment:
Is there a specific reason for this change?

Reply:
yes, it crashed on android cause we're rendering `false`. It's somehow handled on iOS.